app/assets/stylesheets/application.css.scss

```diff
@@ -119,3 +117,65 @@ a:hover {
 #footer {
     background:#000;
 }
+
+input[type="text"], input[type="password"], input[type="number"] {
+	padding: .6em .25em;
+}
+
+input[type="submit"] {
+	border : solid 0px #ffffff;
+	border-radius : 3px;
+	moz-border-radius : 3px;
+	-webkit-box-shadow : 0px 0px 5px rgba(0,0,0,0.4);
+	-moz-box-shadow : 0px 0px 5px rgba(0,0,0,0.4);
+	box-shadow : 0px 0px 5px rgba(0,0,0,0.4);
+	font-size : 1em;
+	color : #ffffff;
+	padding : .5em 2em;
+	background : #6c64db;
+	background : -webkit-gradient(linear, left top, left bottom, color-stop(0%,#6c64db), color-stop(0%,#5b54b8), color-stop(0%,#544e9c), color-stop(0%,#47428a), color-stop(100%,#544e9c));
+	background : -moz-linear-gradient(top, #6c64db 0%, #5b54b8 0%, #544e9c 0%, #47428a 0%, #544e9c 100%);
+	background : -webkit-linear-gradient(top, #6c64db 0%, #5b54b8 0%, #544e9c 0%, #47428a 0%, #544e9c 100%);
+	background : -o-linear-gradient(top, #6c64db 0%, #5b54b8 0%, #544e9c 0%, #47428a 0%, #544e9c 100%);
+	background : -ms-linear-gradient(top, #6c64db 0%, #5b54b8 0%, #544e9c 0%, #47428a 0%, #544e9c 100%);
+	background : linear-gradient(top, #6c64db 0%, #5b54b8 0%, #544e9c 0%, #47428a 0%, #544e9c 100%);
+	filter : progid:DXImageTransform.Microsoft.gradient( startColorstr='#6c64db', endColorstr='#544e9c',GradientType=0 );
+	margin-bottom: 12px;
+}
+
+select {
+	width: 160px;
+	-webkit-appearance: none;
+	padding: 5px 6px;
+	height: 2.7em;
+	overflow: hidden;
+	background: url('dropdown.png') no-repeat right #FEFEFE;
+	border: #ccc 1px solid;
+	-webkit-border-radius: 3px;
+	border-radius: 3px;
+	-webkit-box-shadow: inset 0px 0px 10px 1px #FEFEFE;
+	box-shadow: inset 0px 0px 10px 1px #FEFEFE;
+}
+
+#new_trip {
+	margin-top: 50px;
+}
+
+#new_trip table {
+	margin: 0 auto;
+}
+
+#new_trip td {
+	padding-top: 8px;
+	padding-bottom: 15px;
+}
+
+#new_trip th {
+	padding-top: 25px;
```

Comment:
Line should be indented with spaces, not tabs<br>Properties should be ordered padding-bottom, padding-top